service/raft/group0_state_machine.cc

```diff
@@ -199,16 +199,16 @@ future<> group0_state_machine::load_snapshot(raft::snapshot_id id) {
     _ss._topology_state_machine.event.broadcast();
 }
 
-future<> group0_state_machine::transfer_snapshot(gms::inet_address from, raft::snapshot_descriptor snp) {
+future<> group0_state_machine::transfer_snapshot(gms::inet_address from_ip, raft::server_id from_id, raft::snapshot_descriptor snp) {
```

Comment:
Why do we need both ip and id? The caller translate from id to ip to call the function.

Reply:
I think you meant callee (`group0_state_machine::transfer_snapshot`)? Yes, I see that we can access the raft address from there and perform the translation in that function, so the `from_ip` argument is not needed.

Reply:
Removed the `from_ip` argument, the function now translates the host_id to IP itself. I had to pass the raft address map to `group0_state_machine` for that, though.